cfn-resources/global-cluster-config/cmd/resource/resource.go

```diff
@@ -0,0 +1,376 @@
+package resource
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+
+	"github.com/spf13/cast"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	"github.com/openlyinc/pointy"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var RequiredFields = []string{constants.PubKey, constants.PvtKey, constants.ClusterName, constants.ProjectID}
```

Comment:
The RequiredFields are the same for all the functions?

Reply:
yes